modules/passkey/contracts/interfaces/ICustomSignerFactory.sol

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.8.0 <0.9.0;
+
+/**
+ * @title ICustomECSignerFactory
+ * @dev Interface for creating and verifying ECDSA signers. This is a generalized interface that should be
+ *  compatible with curves of any order size. Currently not used in the project and exists here for reference.
+ */
+interface ICustomECSignerFactory {
```

Comment:
nit, this isn't an EC signer - in particular, it would support an interface to passkey with RSA key.